app/components/submission/ManuscriptMetadata/SubjectAreaDropdown.js

```diff
@@ -0,0 +1,165 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled, { withTheme } from 'styled-components'
+import { th } from '@pubsweet/ui'
+import Select from 'react-select'
+
+const Root = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: ${th('gridUnit')};
+`
+
+const SelectLimitMessage = styled.p`
+  color: ${th('colorSuccess')};
+`
+
+const eLifeOptions = [
+  {
+    value: 'biochemistry-chemical-biology',
+    label: 'Biochemistry and Chemical Biology',
+  },
+  { value: 'cancer-biology', label: 'Cancer Biology' },
+  { value: 'cell-biology', label: 'Cell Biology' },
+  {
+    value: 'chromosomes-gene-expression',
+    label: 'Chromosomes and Gene Expression',
+  },
+  {
+    value: 'computational-systems-biology',
+    label: 'Computational and Systems Biology',
+  },
+  {
+    value: 'developmental-biology-stem-cells',
+    label: 'Developmental Biology and Stem Cells',
+  },
+  { value: 'ecology', label: 'Ecology' },
+  {
+    value: 'epidemiology-global-health',
+    label: 'Epidemiology and Global Health',
+  },
+  { value: 'evolutionary-biology', label: 'Evolutionary Biology' },
+  { value: 'genetics-genomics', label: 'Genetics and Genomics' },
+  {
+    value: 'human-biology-medicine',
+    label: 'Human Biology and Medicine',
+  },
+  {
+    value: 'immunology-inflammation',
+    label: 'Immunology and Inflammation',
+  },
+  {
+    value: 'microbiology-infectious-disease',
+    label: 'Microbiology and Infectious Disease',
+  },
+  { value: 'neuroscience', label: 'Neuroscience' },
+  { value: 'physics-living-systems', label: 'Physics of Living Systems' },
+  { value: 'plant-biology', label: 'Plant Biology' },
+  {
+    value: 'structural-biology-molecular-biophysics',
+    label: 'Structural Biology and Molecular Biophysics',
+  },
+]
+
+const findOptionByValue = (value, options) =>
+  options.find(option => option.value === value)
+
+export class ThemelessSubjectAreaDropdown extends React.Component {
+  constructor(props) {
+    super(props)
+
+    const savedOptions = this.props.savedValues.map(value =>
+      findOptionByValue(value, eLifeOptions),
```

Comment:
i would say this abstraction isn't helpful

Reply:
Do you mean it's not helpful to have it saved in a separate variable called 'savedOptions'? So you would have it directly inside `this.state`?
```
    this.state = {
      selectedOptions: this.props.savedValues.map(value =>
          findOptionByValue(value, eLifeOptions),
      ),
      hasReachedMultiselectLimit: false,
    }
```

Or do you mean that this mapping isn't necessary at all?

Reply:
Here's the reasoning behind adding this to the code at all:

[This is the commit](https://github.com/elifesciences/elife-xpub/pull/182/commits/d3ac5815d07a08aeaa1b52ef1692f97c1f316178) where I added this code in. Beforehand, when a user selected some options, clicked 'Next', then clicked 'Back', their selected options did not appear.

On clicking 'Next', the values were always being saved into the Formik form. So at first I thought I just needed to add a new prop (`savedValues={values.subjectAreas}`) to the ValidatedField component that surrounds the SubjectAreaDropdown - see [this line in that commit](https://github.com/elifesciences/elife-xpub/pull/182/commits/d3ac5815d07a08aeaa1b52ef1692f97c1f316178#diff-bec0d7799dc3aae97e34200382de8504R66)

But this didn't quite work because graphql returns [Object Object] & the user just saw 2 empty blue boxes with no words inside.

That's because I was saving the whole object:
```
{
    value: 'structural-biology-molecular-biophysics',
    label: 'Structural Biology and Molecular Biophysics',
}
```
rather than the value alone.

If we save the value alone in the database, then once we get the values back (e.g. `['structural-biology', 'cancer-biology']`, we need to convert this array back into the object format, containing value & label.

Does any of that make sense?

Reply:
i was referring only to the `findOptionsByValue` func, which isn't reused

Reply:
Ohhhhh! Gotcha!